package.json

```diff
@@ -35,24 +35,23 @@
   "homepage": "https://github.com/localstack/serverless-localstack",
   "dependencies": {
     "adm-zip": "^0.5.10",
-    "aws-sdk": "^2.402.0",
-    "es6-promisify": "^6.0.1"
+    "aws-sdk": "^2.402.0"
   },
   "devDependencies": {
-    "chai": "^4.1.2",
+    "chai": "^4.3.7",
     "chai-string": "^1.4.0",
-    "fs-extra": "^7.0.0",
-    "jasmine": "^3.2.0",
+    "fs-extra": "^11.1.0",
+    "jasmine": "^4.5.0",
     "json2yaml": "^1.1.0",
     "nodemon": "^2.0.20",
-    "rimraf": "^2.6.2",
-    "serverless": "^2.30.0",
-    "sinon": "^6.2.0",
-    "tempy": "^0.2.1",
+    "rimraf": "^4.1.2",
+    "serverless": "^3.27.0",
```

Comment:
⚠️ The major serverless upgrade from v2 to v3 introduces a lot of breaking changes ([upgrading-v3](https://www.serverless.com/framework/docs/guides/upgrading-v3)) and a long list of new deprecation warnings ([deprecations](https://www.serverless.com/framework/docs/deprecations)) also related to [variable resolving](https://www.serverless.com/framework/docs/deprecations#new-variables-resolver).

I agree that we should upgrade to v3. To indicate breaking changes, we should probably bump `serverless-localstack` to a new major version. I wonder whether we should batch this with further breaking changes related to the upcoming LocalStack v2 release 🤔 

* `InvalidLocationConstraint` issue with new S3 provider: https://github.com/localstack/localstack-demo/issues/52
* New default flags for starting LocalStack [here](https://github.com/localstack/serverless-localstack/pull/212/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556L471)
* Related: [localstack-pro-samples](https://github.com/localstack/localstack-pro-samples) still using Serverless v2
  * `serverless-websockets`
  * `serverless-lambda-layers`

On on hand, we should resolve this variable resolver issue quickly.
On the other hand, we should think about how to savely bump the Serverless dependency to v3 and align with the LocalStack v2 release.

@whummer What do you think?